src/config/entities/__tests__/configuration.ts

```diff
@@ -21,6 +21,80 @@ export default (): ReturnType<typeof configuration> => ({
   balances: {
     balancesTtlSeconds: faker.number.int(),
     providers: {
+      safe: {
```

Comment:
Is there a particular reason to name this `safe`? We don't nest providers in other configurations, e.g. `relay`.

Reply:
It was an intent to align with the `BalancesApiManager` strategy and have the providers (`SafeBalancesApi` and `ZerionBalancesApi` under the `balances` tree. 

Regarding the nesting, we could also omit the `providers` key and have just `balances.zerion` and `balances.safe` (or another key instead of `safe` if you find it confusing. What do you think?

Reply:
I'm happy to keep it as. Thank you for the clarification!